test-data/users.data.ts

```diff
@@ -0,0 +1,4 @@
+export const userCredentials = {
+    userEmail: "tomek.pawlak60@gmail.com",
```

Comment:
export const user = {
    email: 'blah',
    password: 'blah2',
};

Reply:
I solved it, but why we do it like that?